src/main/java/com/playmonumenta/libraryofsouls/SoulEntry.java

```diff
@@ -20,27 +33,9 @@
 import org.bukkit.potion.PotionEffect;
 import org.bukkit.potion.PotionEffectType;
 import org.bukkit.util.BoundingBox;
+import org.jetbrains.annotations.Nullable;
 
-import com.goncalomb.bukkit.mylib.reflect.NBTTagCompound;
-import com.goncalomb.bukkit.nbteditor.nbt.EntityNBT;
-import com.goncalomb.bukkit.nbteditor.nbt.variables.BooleanVariable;
-import com.goncalomb.bukkit.nbteditor.nbt.variables.EffectsVariable;
-import com.google.gson.Gson;
-import com.google.gson.JsonArray;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonObject;
-import com.playmonumenta.libraryofsouls.bestiary.BestiaryArea;
-import com.playmonumenta.libraryofsouls.bestiary.BestiaryEntryInterface;
-import com.playmonumenta.libraryofsouls.bestiary.BestiaryManager;
-import com.playmonumenta.libraryofsouls.bestiary.BestiarySoulInventory;
-
-import net.kyori.adventure.text.Component;
-import net.kyori.adventure.text.format.NamedTextColor;
-import net.kyori.adventure.text.format.TextDecoration;
-
-public class SoulEntry implements Soul, SoulGroup, BestiaryEntryInterface {
-	private static Gson gson = null;
-
+public class SoulEntry implements Soul, BestiaryEntryInterface {
```

Comment:
(In case I question this again on future re-reads) Soul extends SoulGroup, no issues here.

Reply:
This one came from my compiler giving me a warning, lol.